### ISSUE TITLE
Special signs should not be taken into account for the letter count

### ISSUE THREAD

**Issue by benintech** [bartekplus/FluentTyper#7]
Per my configuration, the propositions appear only after 5 letters. Yet with a quote the quote doesn't seem to be considered as a word separator, which it is :
![c-est-l-aventure-5-lettres](https://user-images.githubusercontent.com/5140169/183619081-b3432b5c-c02f-419a-9266-2edb08b30a3b.png)

I wanted to type « C'est l'aventure », the word list should only be displayed after the fifth letter of « aventure », e.g. « avent ».

It seems all signs are taken into account. Again, in the following example the opening parenthesis is considered as a letter, it seems :
![punctuation-signs-sould-not-count](https://user-images.githubusercontent.com/5140169/183693157-81ab6c99-bfd0-42c9-ae87-a5102f49be07.png)
The list should only appear after the 5th letter, not after the 4th like here.

**Comment by benintech** [comment]
Works for the first example I gave, in French, « l'avent ».
Doesn't work for the second example though, in English, « (unfo ». The list still pops up after only 4 characters.

I think every non-letter character should be considered as a word separator. I opened a separate issue (#11) for that, because IMO the problem goes beyond the fact of displaying the suggestions after n characters or n-1 characters, which would be minor.
